### PR TITLE
1st draft of fundraising plan

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Comment by hmenager** [comment]
Looks good, a few remarks:
- can (should) we define more precisely what a "featured" presentation is, by opposition to a normal one? I for one wouldn't be able to explain the difference.
- should we mention that promotion of events is conditioned by them obeying the CWL code of conduct?
- is the promotion of CWL job offers from sponsors via mailing list/twitter unconditional?

**Comment by hmenager** [comment]
Also, does this mean that CWL events and jobs promotion is reserved to sponsors, or do we still promote others?

**Comment by mr-c** [comment]
Thanks @hmenager ! I added a change to define "featured" presentations, and to say that promoted events either need to use the CWL code of conduct, or an equivalent CoC.

> is the promotion of CWL job offers from sponsors via mailing list/twitter unconditional?

At this time, yes

> Also, does this mean that CWL events and jobs promotion is reserved to sponsors, or do we still promote others?

No

**Comment by geoffjentry** [comment]
LGTM

File: fundraising.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# CWL Sponsorship Program
+
+## Named Sponsorship Categories
+
+1. Inaugural **Sustaining** Sponsor
+
+   Requires 2 year commitment at levels indicated below, must make
+   commitment prior to December 31th, 2022 (to have the “Inaugural” title)
+   --- only available in 2022/2023. 
+   Rate is only applicable for 2022; additional levels may be added or changed in 2023 onwards.
```
Comment (mr-c) [inline fundraising.md]:
```suggestion
   Rate is only applicable for 2022 and 2023; additional levels may be added or changed in 2024 onwards.
```

**Comment by tetron** [comment]
LGTM

**Comment by bdusenbery** [comment]
Looks good!

**Pull request merged**